### PR TITLE
Reorganize tutorial notebooks

### PR DESCRIPTION
**Pull request by changhiskhan**
Renamed labels => 3. EDA
Renamed peeking_duck => 4. Model inference (symlinked since it's tweeted) Renamed ingestion => 3. Creating Lance Datasets (and also expanded it)

Added "hello world"

This requires the pandas ImageDtype extension so I won't merge this until that's done.

### CODE REVIEW

File: python/lance/tests/types/pandas/test_image.py [hunk 1]
```diff
@@ -0,0 +1,243 @@
+import numpy as np
```
Comment (eddyxu) [inline python/lance/tests/types/pandas/test_image.py:1]: i assume this was based from #267 ?

File: python/test_mac.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```
Comment (eddyxu) [inline python/test_mac.sh:1]: Put into `tools/`?

File: python/lance/tests/types/pandas/test_image.py (hunk 1 above)
Comment (changhiskhan) [reply python/lance/tests/types/pandas/test_image.py:1]: Yeah, it's on top of that one

File: python/test_mac.sh (hunk 2 above)
Comment (changhiskhan) [reply python/test_mac.sh]: Done

**Review by eddyxu — APPROVED** [review]
LGTM

**Pull request merged**